### PR TITLE
Made the secondary CTA hidden when not of any use

### PR DESCRIPTION
**Pull request by hb2708**
This solves #25 

**Changes:**

1. Made the secondary CTA hidden when not of any use
2. Added comments to pauseButton show/hide logic
3. Refactored the code to set the isReset flag value to be more clean and understandable.
As previously it was bit confusing

### CODE REVIEW

**Comment by hb2708** [comment]
@LeeU1911 Take a look when get time.

File: static/js/pomodoro.js [hunk 1]
```diff
@@ -149,6 +150,17 @@ function pauseTimer(){
 }
 
 function updatePauseButton(){
+  /*
+    At this point if isReset
+    Is true Then that mean POMODORO is active so we show the pauseButton
```
Comment (LeeU1911) [inline static/js/pomodoro.js]: Can you fix the comment format a bit? `At this point if isReset is true then that means ...`

**Review by LeeU1911 — CHANGES_REQUESTED** [review]
Approved after a small format change.

File: static/js/pomodoro.js (hunk 1 above)
Comment (hb2708) [reply static/js/pomodoro.js]:
```
 /*
    At this point,
    if isReset is true Then that mean POMODORO is active so we show the pauseButton
    else POMODORO is not active so we don't need the pauseButton so we hide it
   */
```

This looks good??

**Comment by LeeU1911** [comment]
#response_container_BBPPID{font-family: initial; font-size:initial; color: initial;}  Yep, thanks!                                                                                                                                             Sent via BlackBerry Hub+ Inbox for Android                                                                                                                                                                           From: notifications@github.comSent: March 29, 2020 7:18 PMTo: try-pomodoro@noreply.github.comReply-to: reply@reply.github.comCc: imleeu@gmail.com; mention@noreply.github.comSubject: Re: [LeeU1911/try-pomodoro] Made the secondary CTA hidden when not of any use (#26)  
@hb2708 commented on this pull request.



In static/js/pomodoro.js:
> +    At this point if isReset
+    Is true Then that mean POMODORO is active so we show the pauseButton

 /*
    At this point,
    if isReset is true Then that mean POMODORO is active so we show the pauseButton
    else POMODORO is not active so we don't need the pauseButton so we hide it
   */

This looks good??

**Comment by hb2708** [comment]
Okey i have updated the comment

**Review by LeeU1911 — APPROVED** [review]

**Pull request merged**